common.mk

```diff
@@ -32,9 +32,40 @@ CCACHE ?= $(shell which ccache) # Don't remove this comment (space is needed)
 # # Set QEMU_VIRTFS_ENABLE to 'y' and adjust QEMU_VIRTFS_HOST_DIR
 # # Then in QEMU, run:
 # # $ mount -t 9p -o trans=virtio host <mount_point>
+# # Or enable QEMU_VIRTFS_AUTOMOUNT
 QEMU_VIRTFS_ENABLE		?= n
 QEMU_VIRTFS_HOST_DIR	?= $(ROOT)
 
+# Persistent Secure Storage via shared folder
+# # Set QEMU_PSS_ENABLE to 'y' and adjust QEMU_PSS_HOST_DIR
+# # Then in QEMU, run:
+# # $ mount -t 9p -o trans=virtio secure /data/tee
+# # Or enable QEMU_PSS_AUTOMOUNT
+QEMU_PSS_ENABLE			?= n
+QEMU_PSS_HOST_DIR		?= $(ROOT)
+
+ifeq ($(QEMU_PSS_ENABLE),y)
+$(call force,QEMU_VIRTFS_ENABLE,y,required by QEMU_PSS_ENABLE)
```

Comment:
I'm sorry, I mixed up the build environments.
Actually macro `force` is something defined only in optee_os make script, not here in the **build.git** makefiles.

Yet, since this syntax looks nice, I think we could define the macro in this **common.mk** file:
```makefile
# Set a variable or error out if it was previously set to a different value
# The reason message (3rd parameter) is optional
# Example:
# $(call force,CFG_FOO,foo,required by CFG_BAR)
define force
$(eval $(call _force,$(1),$(2),$(3)))
endef

define _force
ifdef $(1)
ifneq ($($(1)),$(2))
ifneq (,$(3))
_reason := $$(_empty) [$(3)]
endif
$$(error $(1) is set to '$($(1))' (from $(origin $(1))) but its value must be '$(2)'$$(_reason))
endif
endif
$(1) := $(2)
endef
```

Reply:
> I'm sorry, I mixed up the build environments.

Me too :-/

> Yet, since this syntax looks nice, I think we could define the macro in this common.mk file:

Agreed.

Reply:
Added it to the `defines, macros...` section of `common.mk`.